cli/commands/resources/project/delete.go

```diff
@@ -37,11 +42,98 @@ func _delete(ctx *cli.Context) error {
 		return err
 	}
 
-	projects, err := projectLib.ListResources()
+	project, schema, err := selectDeletion(ctx)
 	if err != nil {
 		return err
 	}
 
+	repos, err := project.Repositories()
+	if err != nil {
+		return err
+	}
+
+	repoNames := []string{repos.Code.Fullname, repos.Configuration.Fullname}
+
+	msg := formatDeleteConfirm(project.Name, repoNames)
+	if prompts.ConfirmPrompt(ctx, msg) {
+		libraries, websites, err := resources(schema)
+		if err != nil {
+			// use i18n
+			return err
+		}
+
+		if resources := resourceNames(websites, libraries); len(resources) > 0 {
+			repoNames = append(repoNames,
+				prompts.MultiSelect(ctx,
+					prompts.MultiSelectConfig{
+						Field:   "resource",
+						Prompt:  "Remove Resources?",
+						Options: resources,
+					},
+				)...,
+			)
+		}
+
+		prj, err := config.Projects().Get(project.Name)
```

Comment:
Should try deleting the project first and if it passes then remove it from the location